content/python/concepts/time-module/sleep.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Remove all the extra line breaks in the metadata

Reply:
Made all changes after review.